spec/controllers/collections_controller_spec.rb

```diff
@@ -71,6 +72,26 @@
         expect(doc[Solrizer.solr_name(:collection)]).to eq afterupdate.to_solr[Solrizer.solr_name(:collection)]
       end
     end
+
+    context "when setting visibility" do
+      it 'creates a public Collection' do
+        col1 = create(:collection, title: ["Public collection"],
+                                   visibility: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC)
+        expect(col1.visibility).to eq Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC
+      end
+
+      it 'creates an institutional Collection' do
+        col1 = create(:collection, title: ["Institution collection"],
```

Comment:
How about creating a new `:institutional_collection` factory and using that here?